core/src/main/java/com/amplifyframework/core/model/query/Where.java

```diff
@@ -63,6 +69,38 @@ public static QueryOptions id(@NonNull final String modelId) {
                 .paginated(Page.firstResult());
     }
 
```

Comment:
Should `Where.id(...)` be deprecated?

Reply:
Marked Where.id as deprecated and link it to the identifier method. To make identifier method type safe it takes in the Model class as a parameter. The Model Class is not available in the Where.id implementation hence it cannot call the identifier method internally.